include/rmm/mr/default_memory_resource.hpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "device_memory_resource.hpp"
+namespace rmm {
+namespace mr {
+
+/**---------------------------------------------------------------------------*
+ * @brief Get the default device memory resource pointer.
+ *
+ * The default device memory resource is used when an explicit memory resource
+ * is not supplied. The initial default memory resource is a
+ * `cuda_memory_resource`.
+ *
+ * The default memory resource is used for all temporary memory allocation.
```

Comment:
Used by what for all what temporary memory allocation? That's a libcudf detail, not a RMM/memory_resource detail, no?

Reply:
Good point. I'll remove this.